components/support/rc_crypto/nss/src/ec.rs

```diff
@@ -23,8 +23,10 @@ use std::{
 #[repr(u8)]
 pub enum Curve {
     P256,
+    SECP384R1,
```

Comment:
I think you can just name it `P384` since both of these curves are NIST approved

Reply:
done

Reply:
Did you forget to send your changes?